.github/workflows/truffle-ruby-test.yml

```diff
@@ -0,0 +1,155 @@
+name: build-with-truffleruby-head
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: "30 14 * * *"
+
+jobs:
+  ruby-versions:
+    uses: ruby/actions/.github/workflows/ruby_versions.yml@master
+    with:
+      engine: truffleruby
+      min_version: 2.6
+
+  reline:
+    needs: ruby-versions
+    name: >-
+      reline ${{ matrix.os }} ${{ matrix.ruby }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: ${{ fromJson(needs.ruby-versions.outputs.versions) }}
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
With the below exclusions, we'll only run `ubuntu-latest` after all. Maybe let's just update this list to that and remove the excludes?